packages/rxfire/package.json

```diff
@@ -33,7 +33,7 @@
     "test:browser:debug": "karma start --browsers=Chrome --auto-watch"
   },
   "main": "dist/index.node.cjs.js",
-  "browser": "dist/index.cjs.js",
+  "browser": "dist/index.esm.js",
```

Comment:
Does rxfire have a top-level dist/ dir? I couldn't find one after a build. It does have a dist/ and a package.json inside each component subdir.

Reply:
hmm, we don't have a top level one. We should remove them all.